<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQHuntSettings</class>
 <widget class="QWidget" name="QQHuntSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Hunt mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="huntDisabledRB">
        <property name="text">
         <string>Disabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="huntManualRB">
        <property name="text">
         <string>Launch on click</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="HuntAutoRB">
        <property name="text">
         <string>Auto launch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="silentModeCB">
     <property name="text">
      <string>Silent mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Self launched behavior :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="slHuntHiddenRB">
        <property name="text">
         <string>Do not show</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="slHuntSilentRB">
        <property name="text">
         <string>Silent hunt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Maximum huntable objects :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="maxObjSB"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
